e2e/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
+dist/
+package-lock.json
+swa-cli.config.json
```

Comment:
is this a typo?

Reply:
No, this is to keep the e2e/samples and e2e/fixtures folders clean.
Or if there's a typo inside, I don't see it 😆